drivers/modem/modem_cmd_handler.c

```diff
@@ -0,0 +1,488 @@
+/** @file
```

Comment:
Choosing net_buf for such command handler was maybe a bit overkill. You always work with maximum size of buffer anyway.

Reply:
The nicest part of net_buf was the append / pull flexibility.   I can optimize this at a future date with a different max buffer primitive.  Thanks.